external/jenkins/dockerfile/Dockerfile

```diff
@@ -29,33 +29,21 @@ ARG IMAGE_VERSION=latest
 FROM $IMAGE_NAME:$IMAGE_VERSION
 
 # Install test dependent executable files
-RUN apt-get update \
-	&& apt-get -y install \
-	ant \
-	apt-transport-https \
-	ca-certificates \
-	dirmngr \
-	curl \
-	git \
-	make \
-	unzip \
-	vim \
-	wget
-
-# Install jenkins tests dependent
-RUN apt-get update \
-	&& apt-get -y install \
-	maven
-	
+RUN apt-get update; \
+    apt-get install -y --no-install-recommends \
+        git \
+        maven \
+    ; \
+    rm -rf /var/lib/apt/lists/*
+
 ENV  JAVA_TOOL_OPTIONS="-Dfile.encoding=UTF8"
 
 # This is the main script to run jenkins tests
 COPY ./dockerfile/jenkins-test.sh /jenkins-test.sh
 
-# Clone jenkins src to /jenkins
 WORKDIR /
-RUN mkdir testResults
```

Comment:
testResults is used to copy test junit result from the container to jobs host, which is used by jenkins junit plugin to show the test results running by docker. 

https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/external/jenkins/dockerfile/jenkins-test.sh#L51
https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/external/jenkins/playlist.xml#L20

Reply:
Oh your right! My bad. I did not see it was used, must of read quickly.

Reply:
@sophia-guo Fixed in latest push!